server-pro/startup.sh

```diff
@@ -45,12 +45,13 @@ chown rstudio-server:rstudio-server /var/lib/rstudio-launcher/Kubernetes
 su rstudio-server -c 'touch /var/lib/rstudio-launcher/Kubernetes/rstudio-kubernetes-launcher.log'
 
 # Activate License
+RSTUDIO_LICENSE_FILE_PATH=${RSTUDIO_LICENSE_FILE_PATH:-/etc/rstudio-server/license.lic}
```

Comment:
```suggestion
RSW_LICENSE_FILE_PATH=${RSW_LICENSE_FILE_PATH:-/etc/rstudio-server/license.lic}
```
The `RSW` prefix is in flight and won't land for a little while, but that's the current plan for the future